app/controllers/application_controller.rb

```diff
@@ -2,5 +2,10 @@ class ApplicationController < ActionController::Base
 
   protect_from_forgery with: :exception
   before_action :authenticate_user!
+  before_action :admin_status
```

Comment:
Not sure how y'all are in to application-level callbacks, but in apps where there are shared partials, I find it SUPER helpful. It's also a quick way to toggle between admin and public viewing. We're going to end up with more permission levels than just admin, too.

Reply:
Is it time to consider moving this into pundit?